.circleci/config.yml

```diff
@@ -14,18 +14,53 @@ commands:
           fingerprints:
             - "0e:50:82:ad:81:17:a5:31:04:13:fd:6e:c7:72:c9:18"
 
+  halt_if_no_package_changes:
+    description: Stops the build if no packages have been changed.
+    steps:
+      - checkout
+      - when:
+          condition:
+            not:
+              equal: [ "master", << pipeline.git.branch >> ]
+          steps:
+            - run: # Install Lerna
+                name: Install Lerna
+                command: yarn global add lerna danger
+            - run: # Run PR Checks
+                name: Run PR Checks
+                command: yarn danger ci
+                  # Runs 'npx lerna list --since origin/master'
+                  # Pipes the output to 'wc' (word count)
+                  # Returns true if no packages have changed (output of 0)
+                  # Returns false if packages have changed (output > 0)
+                  # Halts the job if true 
+                  #
+                  # Note: Ideally, this would be in an env var
+                  # but they can't be shared across Circle Jobs so we run this command before every job.
+                - run: |
+                  if echo $( (( $(npx lerna list --since origin/master | wc -l) == 0 )) && echo "true" || echo "false" ); then
```

Comment:
Runs 'npx lerna list --since origin/master'
                  Pipes the output to 'wc' (word count)
                  Returns true if no packages have changed (output of 0)
                  Returns false if packages have changed (output > 0)
                  Halts the job if true 
                  
                  Note: Ideally, this would be in an env var
                  but they can't be shared across Circle Jobs so we run this command before every job.